### PR TITLE
feat: Ask for rating on first order

### LINKED ISSUE #2002: Ask for rating after successful ticket order
**Describe the feature you'd like**
Ask for a rating from the user on a successful order

**Would you like to work on the issue?**

- [x] Yes

### PR DESCRIPTION
**Pull request by liveHarshit**
Fixes #2002

<p align="center">
 
</p>

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/order/OrderCompletedFragment.kt [hunk 1]
```diff
@@ -109,6 +118,26 @@ class OrderCompletedFragment : Fragment() {
...
+        val alertDialog = AlertDialog.Builder(requireContext())
+            .setView(layout)
+            .setNeutralButton(getString(R.string.no_thanks)) { _, _ ->
+                preferences.putBoolean(DISPLAY_RATING_DIALOG, false)
+            }.setPositiveButton(getString(R.string.maybe_later)) { _, _ ->
+                preferences.putBoolean(DISPLAY_RATING_DIALOG, true)
+            }.show()
+        layout.rateEventyayButton.setOnClickListener {
+            alertDialog.dismiss()
+            val appPackageName = "com.eventyay.attendee"
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/order/OrderCompletedFragment.kt]: Don't hardcode. Fetch actual

Comment (liveHarshit) [reply app/src/main/java/org/fossasia/openevent/general/order/OrderCompletedFragment.kt]: @iamareebjamal But there is something wrong. The package name is different. In the repo, it is `org.fossasia.openevent.general`
https://github.com/fossasia/open-event-android/blob/91eda72bab0ed86f564ed26d259b7acb1eff7c7a/app/src/main/AndroidManifest.xml#L3 And on play store it is `com.eventyay.attendee` https://play.google.com/store/apps/details?id=com.eventyay.attendee

Comment (iamareebjamal) [reply app/src/main/java/org/fossasia/openevent/general/order/OrderCompletedFragment.kt]: See build.gradle

Comment (liveHarshit) [reply app/src/main/java/org/fossasia/openevent/general/order/OrderCompletedFragment.kt]: Updated

**Pull request merged**